### DISCUSSION TITLE
Перестали загружаться полностью изображения

### DISCUSSION THREAD

**Discussion by webkrit** [wpcraft-ru/wooms#494] [Q&A]
Около месяца не использовал синхронизацию с моим складом, тут решил добавить 40 позиций и соответственно синхронизироваться с сайтом. Ошибок по журналам нет. Товары добавляются, категории тоже, а изображения заливаются неполностью по 312 Б, те вместо 100-150 Кб и так все 40 товаров. Очередь чистил, удалял и снова пробовал запустить, изображения переименовывал, что еще пробовать куда копать? Хост reg ru php 7.4, wordpress обновил

**Comment by Stason077** [comment]
Со вчерашнего дня аналогичная ситуация, 1 в 1
Еще 28 апреля главные изображения товаров выгружались нормально

**Comment by Andrey20211** [comment]
Остатки тоже синхронизируются не корректно сейчас. С 25 апреля стало так.  Я так подозреваю из за того что мой склад переходит на Rest API 1.2, и ввели ограничения на количество запросов по старому протоколу 1.1.  Они об этом пишут у себя в ленте.

**Reply by Stason077** [reply]
Вчера выгружал около 1500 товаров на сайт из МойСклад (Всё в порядке, кроме выгрузки основных изображений)
Заодно сравнил несколько товаров по остаткам, всё корректно отображается

По поводу API, в версии WooMS 7.2 вроде как все обработчики перевели на версию REST API 1.2 (Посмотрел в changelog)

На всякий случай написал в тех. поддержку МойСклад, может проблема API с их стороны. Получил ответ - "Сервис должен работать в штатном режиме"

**Reply by Andrey20211** [reply]
Да, действительно. Посмотрел описание к 7.2. 
но вот такое письмо получил от МойСклад:


Здравствуйте!
--
Вы используете интеграцию с МоимСкладом по JSON API 1.1.Запросы к API_идут с IP_адресов 165.231.18х.ххх; 5.101.17.хх; 176.59.141.ххх; 84.252.134.хх; и другие
С 1 ноября 2023 года мы прекращаем поддерживать JSON API версии 1.1 и переходим на версию 1.2, у которой больше возможностей. Отличия и преимущества — здесь.
До 1 ноября мы будем поэтапно снижать лимиты по количеству запросов для JSON API 1.1. Первое снижение лимитов до 30 запросов в 3 секунды будет 10 апреля.

Здравствуйте!
Вы используете интеграцию с МоимСкладом по JSON API 1.1.
Запросы к API_идут с IP_адресов 165.231.182.62; 5.101.157.75; 176.59.141.180; 84.252.134.5; 176.59.143.56; 138.199.59.216; 169.150.209.163 и другие
С 1 ноября 2023 года мы прекращаем поддерживать JSON API версии 1.1 и переходим на версию 1.2, у которой больше возможностей. Отличия и преимущества — [здесь]
До 1 ноября мы будем поэтапно снижать лимиты по количеству запросов для JSON API 1.1. Первое снижение лимитов до 30 запросов в 3 секунды будет 10 апреля.
